test/adapters/net_http_persistent_test.rb

```diff
@@ -10,8 +10,10 @@ def setup
         if defined?(Net::HTTP::Persistent)
           # work around problems with mixed SSL certificates
           # https://github.com/drbrain/net-http-persistent/issues/45
-          http = Net::HTTP::Persistent.new('Faraday')
-          http.ssl_cleanup(4)
+          if Net::HTTP::Persistent::instance_methods.include?(:ssl_cleanup)
+            http = Net::HTTP::Persistent.new('Faraday')
+            http.ssl_cleanup(4)
+          end
```

Comment:
The comment immediately above this references issues with SSL client certificates, I think the correct thing to do is to call [`reconnect_ssl`](https://github.com/drbrain/net-http-persistent/blob/master/lib/net/http/persistent.rb#L881-L886) to address that.